src/layouts/faq/QuestionSection.jsx

```diff
@@ -97,51 +181,90 @@ export const QuestionSection = (props) => {
     }
   }, [faqId, accordions]);
 
+  const setChips = (chips) => {
+    const chipsId = chips.map((chip) => chip.id);
+    return chips
+      .map((chip) => {
+        if (chip.title) {
+          return `<li><button class="chip ${findFilter(chipsId, chip.id) ? 'active' : ''}" data-id="${chip.id}">${
```

Comment:
I'll report this error from linter:
`This call to findFilter always evaluates to false.`